robot_smach_states/src/robot_smach_states/human_interaction/give_directions.py

```diff
@@ -0,0 +1,249 @@
+# ROS
+import PyKDL as kdl
+import rospy
+import smach
+
+# TU/e Robotics
+from robot_skills.robot import Robot
+from robot_skills.util.entity import Entity
+from robot_skills.util.kdl_conversions import point_msg_to_kdl_vector, VectorStamped
+
+# Robot Smach States
+from robot_smach_states.navigation.navigate_to_symbolic import NavigateToSymbolic
+
+
+class GiveDirections(smach.State):
+    """
+    Robot tells the operator how to get to a certain entity
+    """
+    def __init__(self, robot, entity_designator, radius=1.5):
+        # type: (Robot, any) -> any
+        """
+        Init
+
+        :param robot: (Robot) API object
+        :param entity_designator: (EdEntityDesignator) resolving to the entity the operator wants to go to
+        :param radius: passing closer to an entity than this radius, the entity will be mentioned
+        """
+
+        super(GiveDirections, self).__init__(outcomes=["succeeded", "failed"])
+
+        self._robot = robot
+        self._entity_designator = entity_designator
+        self._radius = radius
+
+    def execute(self, ud):
+
+        # Get the constraints for the global planner
+        nav_contraints = NavigateToSymbolic.generate_constraint(
+            robot=self._robot,
+            entity_designator_area_name_map={self._entity_designator: "near"},
+            entity_lookat_designator=self._entity_designator
+        )
+        if nav_contraints is None:
```

Comment:
Typo!

Reply:
But it is consistent. So code shouldn't break